manifoldConfig.cmake.in

```diff
@@ -1,5 +1,30 @@
-set(MANIFOLD_PAR "@MANIFOLD_PAR@")
+# Compute the installation prefix relative to this file, so
+# we can have the subsequent find_package calls first check
+# for bundled dependencies.
+get_filename_component(_FIND_ROOT "${CMAKE_CURRENT_LIST_FILE}" PATH)
+get_filename_component(_FIND_ROOT "${_FIND_ROOT}" PATH)
+get_filename_component(_FIND_ROOT "${_FIND_ROOT}" PATH)
```

Comment:
are these duplicated lines intended or a mistake?

Reply:
Intended - it's constructing a parent path by stripping off the children.  The CMAKE_CURRENT_LIST_FILE is a couple levels down from the path the script wants, so it does multiple calls to walk "up" the path.

Reply:
I see. Maybe add this as a comment?